CMakeLists.txt

```diff
@@ -209,6 +239,7 @@ if (CONFIG_BUILD_WITH_TFM)
     ${TFM_BL2_ARG}
     ${TFM_IPC_ARG}
     ${TFM_REGRESSION_ARG}
+    ${TFM_ENABLED_PARTITIONS_ARG}
```

Comment:
As mentioned previously, I believe this provides better readability in terms of `argument name` and `value list.
```suggestion
    ENABLED_PARTITIONS ${TFM_ENABLED_PARTITIONS_ARG}
```

Reply:
True, this is better, thanks!